README.md

```diff
@@ -0,0 +1,56 @@
+# Survey MoonBear
+
+ Survey MoonBear provides a new way of survey services.
+
+## Link
+- Github: https://github.com/SurveyMoonBear/SurveyMoonbear_APP
+- Heroku: https://moonbear.herokuapp.com/
+
```

Comment:
Let's make a resources section, with links to ARCHITECTURE.md etc.

Reply:
Also link to DB graph

Reply:
Done